ocaml/xapi-cli-server/cli_frontend.ml

```diff
@@ -2659,6 +2668,24 @@ let rec cmdtable_data : (string * cmd_spec) list =
       ; flags= []
       }
     )
+  ; ( "vm-set-uefi-mode"
+    , {
+        reqd= ["uuid"; "mode"]
+      ; optn= []
+      ; help= "Set a VM in UEFI setup or user mode."
```

Comment:
Maybe indicate that either "setup" or "user" is expected by putting them in (single) quotes.